### DISCUSSION TITLE
[Query] Is whereabout 0.6.1 compatible with k8s 1.27

### DISCUSSION THREAD

**Discussion by lubronzhan** [k8snetworkplumbingwg/whereabouts#363] [Q&A] [answered]
Hi is whereabout compatible with k8s 1.27? Or it doesn't matter which k8s version is using, as long as it's > 1.16 that has daemoenset support

https://github.com/k8snetworkplumbingwg/whereabouts#installing-whereabouts
Thank you

**Comment by maiqueb** [comment, marked as answer]
It should be - we actually need it to be.

Please try it out, if something doesn't work in line as it does to 1.26 it is a bug.

**Reply by lubronzhan** [reply]
Thanks
